.github/workflows/build_playground_backend.yml

```diff
@@ -45,12 +43,14 @@ jobs:
       - name: Prepare Go lint env
         run: "sudo ./playground/backend/env_setup.sh"
       - name: Run Lint
-        run: "golangci-lint run internal/..."
+#        run: "golangci-lint run internal/..."
```

Comment:
can you please remove instead of commenting this? I'll merge for now, but maybe let's remove this in a follow up